src/main/sensors/gyro.c

```diff
@@ -966,7 +966,7 @@ static void checkForYawSpin(gyroSensor_t *gyroSensor, timeUs_t currentTimeUs)
     } else {
 #ifndef SIMULATOR_BUILD
         // check for overflow in the axes set in overflowAxisMask
-        const float gyroYawSpinTriggerRate = gyroConfig()->gyro_yaw_spin_threshold * 16.0f;
+        const float gyroYawSpinTriggerRate = gyroConfig()->gyro_yaw_spin_threshold * 16.4f;
```

Comment:
My point is that there's no need for any multiplier.  The `gyro.gyroADCf[Z]` value is already a degrees/second number.  Not applicable, but if a multiplier was required here then having a hardcoded value for the scale would break support for gyros with different scaling.

Reply:
Well the logs were from a hard coded number of 16000, that triggered the spin detection at 1000 degrees per second, so for some reason these values are here as they are, and they work.  It would be much easier if it was all degrees/second, but it isn't.

Again I just copied the gyro code and it uses hard coded values that are *not* in degrees/second.  My multiplier was needed to convert degrees/second in the CLI threshold value to a value that actually worked.  No idea why it's necessary, but it is.